### ISSUE TITLE
Bug:  Icon names for ingredients are not in german

### ISSUE THREAD

**Issue by CaptainKey3s** [TomBursch/kitchenowl#476]
### Is this urgent?

No

### What parts are affected

Backend

### What is the server version

97

### What is the client version

97

### What platform are you using

Android, iOS, Web

### What's the problem 🤔

I don't know if it is a bug or if it wasn't translated yet.  The icon names for the ingredients are not transleted to german. Every icon still has an english name. But the translation report shows that everything should be  translated to german.
![Icons](https://github.com/user-attachments/assets/bb807632-0a8d-4b09-b934-9f4a7d00bec7)

**Comment by DanteMS** [comment]
Yeah, localization of icon names is much needed for people who don't speak English.
There are a lot of local products or just different names for the items that are already featured in the default items, so there always will be the need to set an icon for some item manually.
